cardano-db/test/test-db.hs

```diff
@@ -1,18 +1,37 @@
 {-# LANGUAGE OverloadedStrings #-}
 
+import           Prelude
+
 import           Test.Tasty (defaultMain, testGroup)
 
 import qualified Test.IO.Cardano.Db.Insert
 import qualified Test.IO.Cardano.Db.Migration
 import qualified Test.IO.Cardano.Db.TotalSupply
 import qualified Test.IO.Cardano.Db.Rollback
 
+import           System.Environment (lookupEnv, setEnv)
+import           System.Directory (getCurrentDirectory)
+import           System.FilePath ((</>))
+
 main :: IO ()
-main =
+main = do
+  -- If the env is not set, set it to default.
+  maybePgPassFile <- lookupEnv "PGPASSFILE"
+  _ <- whenNothing maybePgPassFile $ do
```

Comment:
Why not just a simple `case` on that?

Reply:
I guess that can work here as well, `whenSomething` is pretty standard as well.

Reply:
Nevermind I cherry picked a version of that into `master` since it is completely unrelated to Shelley stuff anyway.